packages/toolkit/src/query/react/module.ts

```diff
@@ -50,18 +45,18 @@ declare module '@reduxjs/toolkit/query' {
         >
           ? QueryHooks<Definitions[K]>
           : Definitions[K] extends MutationDefinition<any, any, any, any, any>
-            ? MutationHooks<Definitions[K]>
-            : never
+          ? MutationHooks<Definitions[K]>
+          : never
       }
       /**
        * A hook that accepts a string endpoint name, and provides a callback that when called, pre-fetches the data for that endpoint.
        */
       usePrefetch<EndpointName extends QueryKeys<Definitions>>(
         endpointName: EndpointName,
-        options?: PrefetchOptions,
+        options?: PrefetchOptions
       ): (
         arg: QueryArgFrom<Definitions[EndpointName]>,
-        options?: PrefetchOptions,
+        options?: PrefetchOptions
```

Comment:
```suggestion
        options?: PrefetchOptions,
```

Reply:
Yeah no idea what kind of formatter is responsible for that 😅